PencilCase/PencilCase/scripts/CustomBindings/ListviewBinding.ts

```diff
@@ -1,11 +1,25 @@
 ﻿class ListviewBinding implements KnockoutBindingHandler {
-    public update(element: any, valueAccessor: () => any, allowBindingAccessor: KnockoutAllBindingsAccessor, viewModel: any, bindingContext: KnockoutBindingContext) {
+    public init(element: any, valueAccessor: () => any, allowBindingAccessor: KnockoutAllBindingsAccessor, viewModel: any, bindingContext: KnockoutBindingContext) {
```

Comment:
Here init is OK. NO need to use update.
Because we use subscription in the method. But not depend on the binding context by update method. So init is good enough.
Actually I tried to use update to avoid  using subscription, but failed due to "$(element).listview("refresh")" code, because element not totally loaded, so refresh failed.